src/main/java/com/example/onlinebookstore/repository/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.example.onlinebookstore.repository;
+
+import com.example.onlinebookstore.model.Role;
+import com.example.onlinebookstore.model.RoleName;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Role getRoleByName(RoleName name);
```

Comment:
Maybe it's better return Optional?

Reply:
I'm not sure, because the role is set by default, and we won't extract null a priori